src/main/resources/static/js/reservation-legacy.js

```
@@ -0,0 +1,146 @@
+  }));
+}
+
+function createInput(type) {
+  const input = document.createElement('input');
+  input.type = type;
+  input.className = 'form-control';
+  return input;
+}
+
+function createActionButton(label, className, eventListener) {
+  const button = document.createElement('button');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'createActionButton'. Consider to delete it or rename it.

'createActionButton' is defined here:


(lint/suspicious/noRedeclare)